python/hsfs/core/statistics_engine.py

```diff
@@ -96,10 +98,15 @@ def profile_transformation_fn_statistics(feature_dataframe, columns):
                 "statistics for. A possible cause might be that you inserted only data "
                 "to the online storage of a feature group."
             )
-        return engine.get_instance().profile(
+        content_str = engine.get_instance().profile(
             feature_dataframe, columns, False, True, False
         )
 
+        # add unique value profile to String type columns
+        return self.profile_unique_values(
```

Comment:
move this to `python.profile` instead, because this only works in python engine.

Reply:
If you call it directly from the class like `StatisticsEngine.profile_unique_values`, you can keep the method static.

Reply:
it works also for spark engine. Spark engine also returns array of unique values with the mothod `get_unique_values`, similar what python is doing and then it assembles deequ style json

Reply:
In spark engine we are using deequ to get unique values, right? I mean we should be able to get unique value in python engine by calling 
```
engine.get_instance().profile(
            feature_dataframe, columns, False, True, True
        )
```

Reply:
Since I just saw the conversation here, I can maybe help to clarify. Currently, the deequ counts the number of unique values, but does not capture the values themselves.

Reply:
@tdoehmen thanks for clarification, thats was also my understanding. @kennethmhc in sprark engine I do the following to get unique values:
```
    def get_unique_values(feature_dataframe, feature_name):
        return feature_dataframe.select(feature_name).distinct().collect()
 ```

Reply:
Got it! Is the unique value being used in `encoder` transformation? `encoder` is using "histogram" right now, need to change it?